user/conditional-builds-stages-jobs.md

```diff
@@ -21,7 +21,9 @@ Build requests that do not match the condition will not generate a build, but wi
 
 ## Conditional Stages
 
-Configure Travis CI to only include stages when certain conditions are met, such as only including the stage on the master branch. Stages that do not match the given condition are silently skipped.
+Configure Travis CI to only include stages when certain conditions are met. Stages that do not match the given condition are silently skipped.
+
+For example, this allows the deploy stage to run only on the master branch:
```

Comment:
I would want `master` be either stylized or quoted. (I prefer the former.)

Reply:
done